lib/vehicle.rb

```diff
@@ -6,13 +6,18 @@ class Vehicle
               :make,
               :model,
               :engine
+  
+  attr_accessor :registration_date, 
```

Comment:
With an attr_accessor, remember you are creating two methods, a reader (a getter) and a setter, so we want to make sure we test both of those in the vehicle_spec. Can we read the registration_date AND can we reset it?